codec-http/src/main/java/io/netty/handler/codec/http/ServerCookieEncoder.java

```diff
@@ -68,6 +69,7 @@ public static String encode(Cookie cookie) {
 
         if (cookie.maxAge() != Long.MIN_VALUE) {
             add(buf, CookieHeaderNames.MAX_AGE, cookie.maxAge());
+            addUnquoted(buf, CookieHeaderNames.EXPIRES, HttpHeaderDateFormat.get().format(new Date(cookie.maxAge() * 1000 + System.currentTimeMillis())));
```

Comment:
> @Scottmitch Not sure I get it. HttpHeaderDateFormat has some fallbacks formatters for handling deprecated formats, but the first one uses GMT.

Do we have a dependency between using GMT and UTC? [currentTimeMillis](http://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis()) is UTC, but it seems like GMT is being used by the date formatters (and `new Date()`)?